src/appleseed/renderer/modeling/project/project.cpp

```diff
@@ -126,6 +127,7 @@ struct Project::Impl
     LightPathRecorder                   m_light_path_recorder;
     std::unique_ptr<TraceContext>       m_trace_context;
     RenderingTimer                      m_rendering_timer;
+    std::unique_ptr<TextureStore>       m_texture_store = nullptr;
```

Comment:
This is `nullptr` by default. No need to do `= nullptr`.

Reply:
Seems like you're right, I'll remove it.